### PR TITLE
zh-translation: /docs/concepts/security/index.md

### PR DESCRIPTION
**Pull request by huntsman-li**
ref: https://github.com/servicemesher/istio-official-translation/issues/1052

### CODE REVIEW

**Comment by huntsman-li** [comment]
@googlebot I signed it!

File: content/zh/docs/concepts/security/index.md [hunk 1]
```diff
@@ -1,232 +1,189 @@
...
+1. `ca.istio.io/override` 命名空间标签：*布尔值*标签，它将覆盖所有其他配置，并强制所有 Citadel 实例定位或忽略命名空间
 
-- Otherwise, if a `ca.istio.io/env: "ns-foo"` label is defined in the service account's namespace, the Citadel instance in namespace `ns-foo` will be used for generating key/cert secrets for workloads in the `ServiceAccount`'s namespace.
+1. [`enableNamespacesByDefault` 安全配置](/zh/docs/reference/config/installation-options/#security-options)：如果在 ServiceAccount 的命名空间上未找到标签，则为默认行为
 
-- Otherwise, set `enableNamespacesByDefault` to `true` during installation. If it is `true`, the default Citadel instance will be used for generating key/cert secrets for workloads in the `ServiceAccount`'s namespace.
+从这三个值中，过程详细的反映的策略行为是：[`Sidecar 注入 Webhook`](/zh/docs/ops/setup/injection-concepts/)
 
-- Otherwise, no secrets are created for the `ServiceAccount`'s namespace.
+- 如果 `ca.istio.io/override` 存在且为 true，则为工作负载生成密钥/证书机密。
```
Comment (gaohuag) [inline content/zh/docs/concepts/security/index.md]: 建议
机密=>secrets

**Comment by rootsongjc** [comment]
Headers with anchors lint failed. Can you help us to fix these errors? @geeknoid 
>> 430 files are free from spelling errors
    ./zh/docs/concepts/security/index.md
       79 | ### Istio 安全与 SPIFFE{#istio-security-vs-spiffe} 
       91 | ## PKI{#pki} 
      100 | ### Kubernetes 方案{#kubernetes-scenario} 
      122 | ### Kubernetes 中的代理节点{#node-agent-in-kubernetes} 
      227 | ### 双向 TLS 认证{#mutual-tls-authentication} 
      460 | #### `ServiceRole`{#servicerole} 
      538 | #### `ServiceRoleBinding`{#servicerolebinding} 
      602 | ### 在普通 TCP 协议上使用 Istio 认证{#using-istio-authorization-on-plain-tcp-protocols} 
>> 8 spelling errors found in 429 files
To learn how to address spelling errors, please see https://istio.io/about/contribute/creating-and-editing-pages/#linting
Running ["LinkCheck", "HtmlCheck", "OpenGraphCheck", "ImageCheck", "ScriptCheck"] on ["./public"] on *.html...

**Comment by geeknoid** [comment]
I guess the spell checker treats the fragment name as normal text and insists it be valid. So to fix this, you need to use words from the dictionary:

- kubernetes should be Kubernetes
- servicerole should be service-role
- servicerolebinding should be service-role-binding
- tcp -> TCP
- tls -> TLS

File: content/zh/docs/concepts/security/index.md [hunk 2]
```diff
@@ -1,232 +1,189 @@
...
+{{< image width="80%" link="./architecture.svg" caption="Istio 安全架构" >}}
 
-- **Mixer** to manage authorization and auditing
+在下面的部分中，我们将详细介绍 Istio 安全功能。
 
-{{< image width="80%" link="./architecture.svg" caption="Istio Security Architecture" >}}
+## Istio 身份{#istio-identity}
 
-In the following sections, we introduce the Istio security features in detail.
+身份是任何安全基础架构的基本概念。在服务到服务通信开始时，双方必须与其身份信息交换凭证以用于相互认证目的。
```
Comment (rootsongjc) [inline content/zh/docs/concepts/security/index.md]: 服务到服务 -> 服务间

File: content/zh/docs/concepts/security/index.md [hunk 3]
```diff
@@ -1,232 +1,189 @@
...
-- To defend against the man-in-the-middle attack, they need traffic encryption.
+- 为了抵御中间人攻击，需要流量加密。
+- 为了提供灵活的服务访问控制，需要双向 TLS 和细粒度的访问策略。
+- 要审核谁在什么时候做了什么，需要审计工具。
 
-- To provide flexible service access control, they need mutual TLS and fine-grained access policies.
+Istio Security 尝试提供全面的安全解决方案来解决所有这些问题。
 
-- To audit who did what at what time, they need auditing tools.
+本页概述了如何使用 Istio 的安全功能来保护您的服务，无论您在何处运行它们。特别是 Istio 安全性可以缓解针对您的数据，端点，通信和平台的内部和外部威胁。
```
Comment (rootsongjc) [inline content/zh/docs/concepts/security/index.md]: 并列的词组请使用顿号。

File: content/zh/docs/concepts/security/index.md [hunk 4]
```diff
@@ -1,232 +1,189 @@
...
-The Istio security features provide strong identity, powerful policy, transparent TLS encryption, and authentication, authorization
-and audit (AAA) tools to protect your services and data. The goals of Istio security are:
+- **深度防御**： 与现有安全系统集成，提供多层防御
 
-- **Security by default**: no changes needed for application code and infrastructure
+- **零信任网络**： 在不受信任的网络上构建安全解决方案
 
-- **Defense in depth**: integrate with existing security systems to provide multiple layers of defense
+请访问我们的[双向 TLS 迁移](/zh/docs/tasks/security/mtls-migration/)相关文章，开始在部署的服务中使用 Istio 安全功能。
+请访问我们的[安全任务](/zh/docs/tasks/security/)，有关使用安全功能的详细说明。
```
Comment (rootsongjc) [inline content/zh/docs/concepts/security/index.md]: 这句话缺个动词

File: content/zh/docs/concepts/security/index.md [hunk 5]
```diff
@@ -1,232 +1,189 @@
...
-   Citadel stores the certificate and key pairs as
-   [Kubernetes secrets](https://kubernetes.io/docs/concepts/configuration/secret/).
+1. Citadel 监视 Kubernetes `apiserver`，为每个现有和新的服务帐户创建 SPIFFE 证书和密钥对。Citadel 将证书和密钥对存储为 [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/)。
 
-1. When you create a pod, Kubernetes mounts the certificate and key pair to the pod according to its service account via
-   [Kubernetes secret volume](https://kubernetes.io/docs/concepts/storage/volumes/#secret).
+1. 创建 pod 时，Kubernetes 会根据其服务帐户通过 [Kubernetes secret volume](https://kubernetes.io/docs/concepts/storage/volumes/#secret) 将证书和密钥对挂载到 pod。
 
-1. Citadel watches the lifetime of each certificate, and automatically rotates the certificates by rewriting the Kubernetes secrets.
+1. Citadel 监视每个证书的生命周期，并通过重写 Kubernetes 秘密自动轮换证书。
```
Comment (rootsongjc) [inline content/zh/docs/concepts/security/index.md]: secret 保留原文吧。

**Pull request merged**